tmt/utils/__init__.py

```diff
@@ -1289,8 +1289,8 @@ def _spawn_process() -> subprocess.Popen[bytes]:
         try:
             process = _spawn_process()
 
-        except FileNotFoundError as exc:
-            raise RunError(f"File '{exc.filename}' not found.", self, 127, caller=caller) from exc
+        except FileNotFoundError as error:
+            raise RunError("File not found.", self, 127, caller=caller) from error
```

Comment:
Hmm, maybe should keep the filename in the error for this one.

Reply:
Ok for me, added it back!